code/controllers/subsystem/starmap.dm

```diff
@@ -450,7 +420,7 @@ SUBSYSTEM_DEF(starmap)
 					if(!prob(f_list[ship_to_spawn]))
 						ship_to_spawn = null
 
-				SSship.create_ship(ship_to_spawn,faction.cname,capital_system)
+				SSship.create_ship(ship_to_spawn,faction.cname,capital_system) */
```

Comment:
What's the point in completely removing some things and commenting others? Just remove it all, we have a code history for a reason.